### PR TITLE
v0.1.1 release sync

### PR DESCRIPTION
**Pull request by bekauz**
incorporates the astroport tokenfactory-based liquid pooler contract into the main branch

### CODE REVIEW

File: contracts/astroport-tf-liquid-pooler/README.md
Comment (stiiifff) [inline contracts/astroport-tf-liquid-pooler/README.md:1]: 👌

File: contracts/astroport-tf-liquid-pooler/README.md [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
+workspace, this contract is currently tested on its dedicated
+[v0.1.1 release branch](https://github.com/timewave-computer/covenants/tree/release/v0.1.1).
+
+The local-interchaintest directory there contains the existing e2e tests
+that are modified to use this contract code id instead of the old one.
+
+To ensure that the tests are working as expected, make sure to confirm the
+checksum between the contract that results from wasm optimization in this repo
+and the one in `release/v0.1.1` matches.
+
```
Comment (stiiifff) [inline contracts/astroport-tf-liquid-pooler/README.md:25]: 👍

**Review by stiiifff — APPROVED** [review]

File: contracts/astroport-tf-liquid-pooler/.cargo/config [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[alias]
+wasm = "build --release --lib --target wasm32-unknown-unknown"
+schema = "run --bin schema"
```
Comment (keyleu) [inline contracts/astroport-tf-liquid-pooler/.cargo/config:3]: renaming this to config.toml and modifying 'schema' by name of contract would make it consistent with main (and not get warning)

**Comment by keyleu** [comment]
One question, right now in this state the contract would not be compatible with covenants in main branch because it doesn't have the op_mode, this will be done later I guess?

**Comment by stiiifff** [comment]
> One question, right now in this state the contract would not be compatible with covenants in main branch because it doesn't have the op_mode, this will be done later I guess?

Yes, we will update it with the `op_mode`. I created an [issue](https://github.com/timewave-computer/covenants/issues/318) for that.

**Comment by stiiifff** [comment]
@bekauz @keyleu Anything missing or this can be merged ?

**Pull request merged**